src/package.xml

```diff
@@ -1184,10 +1191,10 @@
         <name>CustomLabel</name>
     </types>
     <types>
-        <name>CustomLabel</name>
+        <members>CustomLabels</members>
+        <name>CustomLabels</name>
     </types>
     <types>
-        <members>Account</members>
```

Comment:
are we sure these should be removed from package.xml?  might be an issue with contributor tool.  same comment for the other standard objects deleted in this file.